src/routes/prototype-routes.ts

```diff
@@ -803,6 +803,11 @@ export async function renderResultsPage(
               },
           ];
 
+    //Remove explanation and suggestions from json editor
+    const maskedJson = { ...prototypeData.json };
```

Comment:
It turns out that use of the spread operator causes lots of internal MongoDB/Mongoose fields to be included. Because it's technically a MongoDB document, we have to convert it to an object first. Note that `import { Document } from 'mongoose';` will need to be added at the top of the file.

See https://stackoverflow.com/questions/48014504/es6-spread-operator-mongoose-result-copy
```suggestion
    const maskedJson = {
        ...((prototypeData as unknown as Document).toObject() as IPrototypeData)
            .json,
    };
```